src/Tribe/iCal.php

```diff
@@ -413,13 +413,24 @@ public function generate_ical_feed( $post = null, $echo = true ) {
 		$content .= "METHOD:PUBLISH\r\n";
 
 		/**
-		 * Allows for customizing the value of the generated iCal file's "X-WR-CALNAME:" property.
+		 * Controls if the property X-WR-CALNAME is added to the file or not, allowing flexibility to disable the option
+		 * as in some cases like users of Outlook does not need this feature, to create a new calendar.
 		 *
-		 * @param string $blogName The value to use for "X-WR-CALNAME"; defaults to value of get_bloginfo( 'name' ).
+		 * @since TBD
+		 *
+		 * @param boolean Controls if add or not the the X-WR-CALNAME property
 		 */
-		$x_wr_calname = apply_filters( 'tribe_ical_feed_calname', $blogName );
+		if ( apply_filters( 'tribe_events_ical_add_calname', true ) ) {
```

Comment:
Instead of having two filters, allow people to remove `X-WR-CALNAME` by making it a `null || false` value.